### DISCUSSION TITLE
Line Width is so small.

### DISCUSSION THREAD

**Discussion by junaidsafdar** [hlorus/CAD_Sketcher#446] [Announcements]
How I can make cad sketcher lines wider or at least 2px because they are too thin.

**Comment by kesselborn** [comment]
I have the same problem (running on macos, Blender 4.0.2, CAD Sketcher 0.27.3). I upped the Entity Scale to 3.00, but that only increases the points, not the lines. 

<img width="200" alt="image" src="https://github.com/hlorus/CAD_Sketcher/assets/12173/8a530c52-2adf-41ef-9a89-403236a331ff">

Note, that on the screenshot, the two points are connected vertically which is hardly visible

**Comment by otecmichal** [comment]
I have the same issue. Entity scale increases the size of points, but not lines. Mac OS, Blender 4.0.2, CAD Sketcher 0.27.3.

**Comment by AdeAnima** [comment]
Same issue here

**Comment by maxgt1** [comment]
I have the same issue.

**Comment by PedroSC100** [comment]
I have the same issue, MAC with apple  silicone and Blender from 4.2  to  4.4
